core-primitives/node-api/metadata/src/pallet_system.rs

```diff
@@ -1,37 +0,0 @@
-/*
```

Comment:
Why did you delete this? Downstream forks might want to use it.

Reply:
Oh I had thought This was all my code this was a mistake lol great catch....